src/components/Calculator.js

```diff
@@ -1,29 +1,43 @@
+/*eslint-disable*/
```

Comment:
- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿